web/src/routes/(user)/albums/[albumId]/+page.svelte

```diff
@@ -448,14 +461,14 @@
             {/if}
 
             {#if album.assetCount > 0}
+              <CircleIconButton title="Slideshow" on:click={handleStartSlideshow} icon={mdiPresentationPlay} />
```

Comment:
This one line being moved has nothing to do with the rest of the PR. Let me know if I should separate it into its own PR.

Issue: the album slideshow is only available to album owners because the button is under the [...] button which is owner only

I don't think this makes sense, I see two solutions:
- make the [...] button available to everyone and make the specific options underneath (such as delete) owner only
- move the slideshow out from under the [...] button.

This is the latter.

Reply:
I moved this to #8757